.github/workflows/docker.yml

```
@@ -1,24 +1,68 @@
 # This workflow will upload a Docker container
 
-name: Upload Docker Container
+name: Build/Upload Docker Container
 
 on:
   workflow_dispatch:
   push:
     branches:
-      - 'master'
+      - '*'
```

I don't think we want to run this on all branches, just PRs and master.

```suggestion
      - 'master'
```

---

OK! I changed that one.